_inc/client/components/footer/index.jsx

```diff
@@ -90,6 +97,25 @@ export class Footer extends React.Component {
 			return '';
 		};
 
+		const maybeShowModules = () => {
+			if ( this.props.userCanManageOptions ) {
```

Comment:
It may be good to add an additional set of conditions here. The modules list is available if:
- You are an admin (you already check that).
- The site is either in Jetpack development mode or connected to WordPress.com.

On a brand new site that is not connected to WordPress.com yet, we should not display that link imo.

You should be able to bring in `this.props.siteConnectionStatus` to get that information:
https://github.com/Automattic/jetpack/blob/fe5b739531bba42eb7fc5724471410694cea9781/_inc/client/state/connection/reducer.js#L122

Reply:
Fixed in 10276a8, thanks for the assistance @jeherve!